images/wire/tests/main.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "help" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME help"
```

Comment:
Is there a more thorough test we can run? I don't know anything about Wire but maybe getting an example app up?

Reply:
Tried here, it's a bit weird but shows it I guess?